docs/stepbystep.md

```diff
@@ -64,48 +64,55 @@ work as well (previous versions could also work, but also may not, so we encoura
 To enhance readability all the commands will be executed as root. To use other users, give it the appropriate permissions
 and use `sudo` as usual (installation from the RPM package creates a special user for the agent, but it will not be
 used along this tutorial).
- 
-## <a name="singledevice">Provisioning a single device with the default API Key</a>
+
+## Provisioning a single device with the default API Key
```

Comment:
Now the index link to this subsection does not work

Reply:
Fixed - facd80d - ToC regenerated